.travis.yml

```diff
@@ -3,38 +3,45 @@ language: python
 matrix:
   include:
     - python: 2.6
```

Comment:
The Python 2.6 build on travis looks like it's not working due to idna only being available for Python 2.7+. We probably need to pin some dependency versions for Python 2.6.

Reply:
Oh, that was the coveralls install; maybe that's why you moved the installation to after test?

Reply:
I moved the coveralls install as a time optimization since it is only necessary on a successful built. I have not looked much into the ptyhon 2.6 issues yet. Note that the PR in the fork has a good number of warnings but does not fail (https://github.com/itziakos/haas/pull/1/checks?check_run_id=109140674).